pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -101,7 +123,25 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 	for _, item := range backups.Items {
 		if item.Name == defaultBackupCRName {
 			defaultFound = true
-			c.backupVarGetter.SetBackupSpec(&item.Spec.EtcdBackupSpec)
+
+			_, err := c.kubeClient.AppsV1().DaemonSets(operatorclient.TargetNamespace).Get(ctx, backupServerDaemonSet, v1.GetOptions{})
+			if err != nil {
+				if !apierrors.IsNotFound(err) {
+					return fmt.Errorf("PeriodicBackupController could not retrieve [defaultBackupDeployment]: %w", err)
+				}
+
+				endpoints, err := getEtcdEndpoints(ctx, c.kubeClient)
+				if err != nil {
+					return fmt.Errorf("PeriodicBackupController failed to list etcd-endpoints config-map: %w", err)
+				}
+
+				etcdBackupDS := deployBackupServerDaemonSet(item, endpoints)
+				_, err = c.kubeClient.AppsV1().DaemonSets(operatorclient.TargetNamespace).Create(ctx, etcdBackupDS, v1.CreateOptions{})
+				if err != nil {
+					return fmt.Errorf("PeriodicBackupController could not create [defaultBackupDeployment]: %w", err)
+				}
+			}
+			klog.V(4).Infof("PeriodicBackupController created DaemonSet [%v] successfully", backupServerDaemonSet)
```

Comment:
isn't this missing a DS update when a customer changes the schedule?

Reply:
plus it will also log every time the loop comes around with a successful GET

Reply:
> isn't this missing a DS update when a customer changes the schedule?

u mean changes to the CR itself, alright need to add this 👍🏽

Reply:
> plus it will also log every time the loop comes around with a successful GET

Good catch 👍🏽 thxxx

Reply:
> isn't this missing a DS update when a customer changes the schedule?

added to `ToDo` list above

> plus it will also log every time the loop comes around with a successful GET

fixed 👍🏽